pom.xml

```diff
@@ -504,22 +504,59 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.2</version>
-                <configuration>
-                    <skipTests>${skipUTs}</skipTests>
-                    <forkCountTests>${forkCountTests}</forkCountTests>
-                    <forkCount>${forkCountTests}</forkCount>
-                    <reuseForks>true</reuseForks>
-                    <excludes>
-                        <exclude>**/*IT.java</exclude>
-                        <exclude>${someModule.test.excludes}</exclude>
-                        <exclude>**/integrationtests/*</exclude>
-                    </excludes>
-                    <includes>
-                        <include>${someModule.test.includes}</include>
-                    </includes>
-                    <parallel>classes</parallel>
-                    <threadCount>5</threadCount>
-                </configuration>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <skip>true</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>parallel-tests</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <skipTests>${skipUTs}</skipTests>
+                            <forkCount>${forkCountTests}</forkCount>
+                            <reuseForks>true</reuseForks>
+                            <parallel>classes</parallel>
+                            <threadCount>5</threadCount>
+                            <includes>
+                                <include>${someModule.test.includes}</include>
+                            </includes>
+                            <excludes>
+                                <exclude>${someModule.test.excludes}</exclude>
+                                <exclude>**/*IT.java</exclude>
+                                <exclude>**/integrationtests/*</exclude>
+                                <exclude>**/TestTTLRunner.java</exclude>
+                                <exclude>**/TestSubscriptionNotificationRunner.java</exclude>
+                                <exclude>**/TestScalingAndFailoverRunner.java</exclude>
+                                <exclude>**/FlowTestTestExecution.java</exclude>
+                                <exclude>**/SingleEventAggregationTest.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
+                    <execution>
```

Comment:
It probably would be good to put the reason behind splitting out these tests into separate runs somewhere. E.g. if a developer writes a new test that should be added to the sequential run, that developer needs to know where to put it.

Reply:
I added a description.